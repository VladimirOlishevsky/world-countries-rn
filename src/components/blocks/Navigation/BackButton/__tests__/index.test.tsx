import React from 'react';
import { BackButton } from '..';
import { render } from '@testing-library/react-native';

describe('BackButton', () => {
  it('BackButton render', () => {
    const country = render(<BackButton />);
    expect(country).toMatchSnapshot()
  });
});
