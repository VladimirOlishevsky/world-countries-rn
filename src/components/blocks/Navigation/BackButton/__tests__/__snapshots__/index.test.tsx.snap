// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackButton BackButton render 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "height": NaN,
      "justifyContent": "center",
      "opacity": 1,
      "width": NaN,
    }
  }
>
  <Icon
    color="#ff9f0a"
    name="chevron-left"
  />
</View>
`;
